pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -95,6 +95,9 @@ type ClusterSpec struct {
 
 	// A spec for mgr related options
 	Mgr MgrSpec `json:"mgr,omitempty"`
+
+	// Remove the OSD that is out and safe to remove only if this option is true
+	RemoveOSDsIfOutAndSafeToRemove bool `json:"removeOSDsIfOutAndSafeToRemove"`
```

Comment:
- Please add an example to cluster.yaml. Maybe just set it to false, and add a couple lines of comments describing the setting.
- Let's add the setting to documentation in the ceph-cluster-crd.md.